seqr/views/utils/terra_api_utils.py

```diff
@@ -91,6 +88,9 @@ def _get_call_args(path, headers=None, root_url=None):
 
 
 def _safe_get_social(user):
+    if not google_auth_enabled() or not hasattr(user, 'social_auth'):
+        return False
```

Comment:
Return `None` seems better than `False` since the purpose of the helper function is to get the social auth information.